patches/cli/0001-Add-RuntimeFrameworkVersion-to-build-with-source-bui.patch

```diff
@@ -0,0 +1,51 @@
+From 5022a1f4c0f490e81710630e775e496ad7cfd112 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Thu, 23 Aug 2018 11:36:10 -0500
+Subject: [PATCH] Add RuntimeFrameworkVersion to build with source-built
+ MS.NETCore.App under 2.1.401.
```

Comment:
I think it'd be useful for the future to say where `3.0.0-preview1-26529-01` comes from in the patch desc--I'm guessing core-setup.proj's build id?

Reply:
Added description.